Introduction

LIttleAncientForestKami opened the pull request:
per
http://campus.codeschool.com/courses/mastering-github/level/1/section/2/forking-and-cloning

Peer code reviews are the single biggest thing you can do to improve your code.
      -- Jeff Atwood

LIttleAncientForestKami committed:
Introduction
per
http://campus.codeschool.com/courses/mastering-github/level/1/section/2/forking-and-cloning

Peer code reviews are the single biggest thing you can do to improve your code.
      -- Jeff Atwood

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1 @@
+Hi, I'm LAFK.
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

LIttleAncientForestKami committed:
With path now, as requested
in PR comment

There is no programming language–no matter how structured–that will prevent programmers from making bad programs.
      -- Larry Flon

LIttleAncientForestKami commented:
Sure. Done.

LIttleAncientForestKami committed:
Fave path in many ways
Cause first didn't work

Ostatecznie o sprawności działania nowego systemu zadecyduje realne działanie.
		 -- BIP miasta Kraków

LIttleAncientForestKami committed:
Skills updated

The more I C, the less I see.

LIttleAncientForestKami committed:
List
made out of skills

Measuring programming progress by lines of code is like measuring aircraft building progress by weight.
      -- Bill Gates

LIttleAncientForestKami committed:
History kill as a skill
as per request

The real design action is in the memory sub-system - caches, buses, bandwidth and latency
      -- Richard Sites (DEC Alpha Architect)

codeschool-kiddo committed:
Added new rule about coffee

LIttleAncientForestKami committed:
Rules spelling fixes
Coffee, dojo

C++ : Where friends have access to your private members.
      -- Gavin Russell Baker

LIttleAncientForestKami committed:
Merge pull request #2 from LIttleAncientForestKami/new_rules

Too many people are spilling coffee - merge via github

LIttleAncientForestKami committed:
Merge branch 'master' of github.com:LIttleAncientForestKami/dojo_rules
